python/tank_vendor/shotgun_authentication/errors.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+All custom exceptions that this module emits are defined here.
+"""
+
+
+class AuthenticationModuleError(Exception):
```

Comment:
is `ShotgunAuthenticationError` clearer? The `Module` part of the exception doesn't seem to add anything in terms of readability but imagine you run this code from Maya. In that case, it would probably be useful to know that this came from `Shotgun` and not from somewhere else.